docs/cubeviz/plugins.rst

```diff
@@ -326,8 +326,8 @@ have valid flux units. For 3D data, the current :ref:`slice` is used.
 
 .. _cubeviz-export-plot:
```

Comment:
should this reference name be changed from _cubeviz-export-plot to cubeviz-export (in this and in {otherconfigs)/plugins.rst?)

Reply:
I wasn't sure, I didn't want to break existing RTD links so left them alone, but maybe we should put in redirects.... @pllim - what's the best path forward here?

Reply:
The reference name isn't user facing, I would just leave it so existing intersphinx won't break.